src/Google.Api.Gax/Api/Gax/IScheduler.cs

```diff
@@ -0,0 +1,44 @@
+﻿/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file or at
+ * https://developers.google.com/open-source/licenses/bsd
+ */
+
+using System;
+using System.Runtime.CompilerServices;
```

Comment:
I think that was when we were experimenting with TaskAwaitable (or whatever). Will zap.